cloud/cluster/memory/node.go

```diff
@@ -0,0 +1,48 @@
+package memory
+
+import (
+	"fmt"
+	"github.com/scootdev/scoot/cloud/cluster"
+)
+
+type idNode struct {
+	id cluster.NodeId
+}
+
+func (n *idNode) Id() cluster.NodeId {
+	return n.id
+}
+
+func (n *idNode) String() string {
+	return string(n.id)
+}
+
+func NewIdNode(id string) cluster.Node {
+	return &idNode{cluster.NodeId(id)}
+}
+
+var _ cluster.Node = (*idNode)(nil)
+
+func NewIdNodes(num int) []cluster.Node {
```

Comment:
Ohhhhhh. Yup, you're right.

You can create that set of nodes by using the function above, you just don't get the helper function that creates the whole slice.